spyder/config/main.py

```diff
@@ -146,12 +146,14 @@
              {
               'default': True,
               'custom': False,
+              'custom_conda': False,
               'umr/enabled': True,
               'umr/verbose': True,
               'umr/namelist': [],
               'custom_interpreters_list': [],
               'custom_interpreter': '',
-              'last_envs': {}
+              'last_envs': {},
+              'conda_path': ''
```

Comment:
```suggestion
              'conda_path': '',
```

Since we're touching this line anyway, may as well use our modern, easier-to-maintain code style.